### PR TITLE
Update ES translations: capa convectiva

### PR DESCRIPTION
**Pull request by Boran**
Describe the changes you have conducted here

Preview the messages on https://inlang.com/github.com/Boran/soaringmeteo_transDE .

### CODE REVIEW

**Comment by julienrf** [comment]
Could you please try to sync your `main` branch with the `main` branch of this repo before writing your changes?

**Comment by julienrf** [comment]
See the guide here: https://github.com/soaringmeteo/soaringmeteo/blob/main/TRANSLATING.md#fix-or-complete-an-existing-translation

**Comment by Boran** [comment]
> Could you please try to sync your `main` branch with the `main` branch of this repo before writing your changes?

Did a conflict merge for esx.json from the UI, should be ok now. On the command line it got complicated with conflicts on frontend/messages/it.json. Boran:main was too old.

File: frontend/messages/es.json [hunk 1]
```diff
@@ -45,18 +45,18 @@
   "layerCloudsAndRainLegend": "las nubes y la lluvia",
   "layerCumulusDepth": "Nubes cúmulos",
   "layerCumulusDepthLegend": "el espesor de los cúmulos",
-  "layerSoaringLayerDepth": "Espesor de la capa límite",
-  "layerSoaringLayerDepthLegend": "el espesor de la capa límite (convectiva)",
+  "layerSoaringLayerDepth": "Espesor de la capa convectiva",
```
Comment (julienrf) [inline frontend/messages/es.json:48]: Is it possible to shorten it? E.g. “Espesor capa convectiva”?

File: frontend/messages/es.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
   "summaryTotalCloudCover": "Nubosidad total",
   "summaryRainfall": "Lluvia",
   "summaryCumuliDepth": "Profundidad de los cúmulos",
-  "summarySoaringLayerDepth": "Espesor de la capa límite",
+  "summarySoaringLayerDepth": "Espesor de la capa convectiva",
```
Comment (julienrf) [inline frontend/messages/es.json:71]: Same here, does it work to remove the articles? “Espesor capa convectiva”

**Pull request merged**